### ISSUE TITLE
train new model failed. but no error message

### ISSUE THREAD

**Issue by cy163** [crownpku/Information-Extraction-Chinese#62]
Hello,

 I tried to establish  a new model by running  "python3 train_GRU.py".  Before doing that, I execute "sudo  mv ./model ./model-origin" to keep the original model. And create a new 'model' folder by "sudo mkdir ./model". However, it failed. I could see nothing under the ./model folder after running the program.
To debug the program, I inserted some print lines in the code just as follows. 
It printed out 'I am here-7, but did not 'I am here-8'. Would you please tell me the reason. I am new to python, tensorflow,etc. BTW,  there was no error reported as running the program.. I am using the CPU version, instead of GPU version tensorflow-1.7. Probably, is that the reason?!

I am using
.......
 print('I am here-7')
                if step % 50 == 0:
                   print('I am here-8')
                    tempstr = "{}: step {}, softmax_loss {:g}, acc {:g}".format(time_str, step, loss, acc)
                    print(tempstr)

            print('I am here-8')

WARNING:tensorflow:From /Information-Extraction-Chinese-master/RE_BGRU_2ATT/network.py:132: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
